Content.Shared/Damage/Systems/DamageableSystem.cs

```diff
@@ -386,6 +388,49 @@ public void SetAllDamage(EntityUid uid, DamageableComponent component, FixedPoin
             // Shitmed Change End
         }
 
+        /// <summary>
+        ///     Change the DamageContainer of a DamageableComponent.
+        /// </summary>
+        public void ChangeDamageContainer(EntityUid uid, string newDamageContainerId, DamageableComponent? component = null)
+        {
+            if (!Resolve(uid, ref component, logMissing: false)
+            || newDamageContainerId == component.DamageContainerID)
+            {
+                return;
+            }
```

Comment:
```suggestion
                return;
```
cmon guys the brackets